CHANGELOG.md

```diff
@@ -213,22 +224,38 @@ Fixed
 - Relax version requirements for pdbedit, also handle Debian branding in the version string. (#56553)
 - Fixed indentation error on `cmd.run` orchestration output. (#56554)
 - Fixed issue with getting incorrect SELinux context. (#56557)
+- Fix `file.get_selinux_context` with directories (#56558)
 - Fixed bug updating boot parameters with `virt`. (#56562)
 - Correctly handle `pymysql.err.InternalError` in `mysql` module. (#56570)
 - Fixed `panos` commit example in docs. (#56581)
 - Fixed issue with `salt.utils.functools.call_functions` not checking for expected arguments. (#56584)
 - Fixed a broken statement when using arbitrary `kwargs` in mine.value. (#56593)
+- Correct formatting issue with highstate output module (#56598)
 - Fixed support for booting VMs with UEFI on virt. (#56613)
+- Check kwargs for expected arguments in salt.utils.functools.call_function (#56635)
 - Updated old redirects and http->https fixes in docs. (#56655)
 - Renamed `salt/utils/docker/` to `salt/utils/dockermod/` to avoid clashes with the `docker` package from pypi. (#56669)
 - Changed behavior to implicitly ignore package epochs and just use the latest one. (#56681)
+- Fix btrfs test on Windows (#56688 - Master port of #52147)
```

Comment:
Same here - no need to call out master port. Just put the original issue # if it exists, if there's no issue, then the original PR.

If there are multiple issues fixed in one PR, then I put the PR #.